etl_manager/meta.py

```diff
@@ -644,6 +645,32 @@ def create_glue_database(self, delete_if_exists=False):
             glue_table_def = tab.glue_table_definition(self.s3_database_path)
             _glue_client.create_table(DatabaseName=self.name, TableInput=glue_table_def)
 
+    def update_glue_database(self, update_database_metadata=True, update_tables_if_exist=False):
+
+        """
+        Updates a database in Glue based on the database object calling the method function.
+        """
+        db = {"Name": self.name, "DatabaseInput": {"Description": self.description, "Name": self.name}}
+        
+        if update_database_metadata:
+            _glue_client.update_database(**db)
+
+        for tab in self._tables:
+            try:
+                _glue_client.get_table(Name=tab.name, DatabaseName=self.name)
+                table_exists = True
+            except _glue_client.exceptions.EntityNotFoundException:
+                table_exists = False
+
+            if not table_exists:
```

Comment:
Might be worth doing the below to reduce duplicate code
```python
if (not table_exists) or (table_exists and update_tables_if_exist):
```

Reply:
they two different things methods on the `_glue_client` - one calls `create_table` and the other `update_table`